core/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -599,7 +599,6 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeLong(autoGeneratedTimestamp);
         if (out.getVersion().after(Version.V_5_3_0_UNRELEASED)) { // TODO update to onOrAfter after backporting
             if (contentType != null) {
-                out.writeBoolean(true);
                 out.writeOptionalWriteable(contentType);
             } else {
                 out.writeBoolean(false);
```

Comment:
hehe I assume there was a failing test around that as read and write were not always matching  :)